pymc_extras/statespace/models/structural/components/level_trend.py

```diff
@@ -189,11 +189,7 @@ def populate_component_properties(self):
             shock_base_names = [
                 name for name, mask in zip(name_slice, self.innovations_order) if mask
             ]
-            self.shock_names = [
-                f"{name}[{obs_name}]"
-                for obs_name in self.observed_state_names
-                for name in shock_base_names
-            ]
+            self.shock_names = [f"{name}" for name in shock_base_names]
```

Comment:
```suggestion
            self.shock_names = shock_base_names
```
If shock_base_names isn't  used anywhere else, just use that list comprehension to directly define `self.shock_names`